Structure_Engine/Modify/SetMaterial.cs

```diff
@@ -33,6 +36,10 @@ public static partial class Modify
         /**** Public Methods                            ****/
         /***************************************************/
 
+        [Description("Sets the material of a Bar by updating the material of its SectionProperty.")]
+        [Input("bar", "The Bar to set the material to, i.e. the Bar to have the Material of its SectionProperty updated.")]
```

Comment:
Material

Reply:
Changing to consistent lower-casing instead, as it is not referring to the physical Material class